Assets/LeapMotion/Core/Scripts/LeapImageRetriever.cs

```diff
@@ -386,14 +390,14 @@ public class EyeTextureData {
     }
 
     public void ApplyCameraProjectionValues(Camera camera) {
```

Comment:
Looks like this file might have been in the middle of some cleanup?  Looks like ApplyCameraProjectionValues and friends can all be removed?

Reply:
Yep good catch. That was some of the first work I did on collapsing the hierarchy. Removed that stuff now